package.json

```diff
@@ -50,13 +50,15 @@
   },
   "homepage": "https://github.com/nthparty/opaque#readme",
   "dependencies": {
-    "libsodium-wrappers-sumo": "0.7.6",
-    "oprf": "2.0.0"
+    "libsodium-wrappers-sumo": "0.7.8",
+    "oprf": "^2.0.0"
```

Comment:
This package should be reinstalled with the `-E` option. My aim with pinning exact versions generally is to avoid supply-chain attacks (such as the sort [proliferated by `node-ipc` not long ago](https://snyk.io/blog/peacenotwar-malicious-npm-node-ipc-package-vulnerability/)). The same should probably be done with `eslint` and `"@typescript-eslint/eslint-plugin` for consistency, if those packages are to remain.